node.js

```diff
@@ -49,7 +49,42 @@ module.exports = {
     'space-before-function-paren': off,
     'keyword-spacing': error,
     'eol-last': [ error, always ],
-    'brace-style': [ error, '1tbs', { allowSingleLine: true } ],
-    'curly': [ error, all ]
+    'brace-style': [ error, '1tbs', { allowSingleLine: false } ],
+    'curly': [ error, all ],
+    'block-spacing': [ error,  always ],
```

Comment:
Enforce spacing after a block starts (if it's on a single line)
https://eslint.org/docs/rules/block-spacing
```js
// Old
function foo() {return true;}
// New
function foo() { return true; }
```

Reply:
A suggestion: no one-liner for blocks, to have a consistent way of defining functions
```javascript
function foo() {
  return true;
}

or 

const foo = () => true // if you really need a one-liner
```

?